starknet/src/testing/proofs/storage.cairo

```diff
@@ -1027,3 +1027,37 @@ pub fn PROOF_2() -> StorageProof {
         value: 0x07a8f1298f7eaa479401,
     }
 }
+
+pub fn PROOF_anvil() -> StorageProof {
```

Comment:
Are this proofs actually taken from anvil?

Reply:
yeah, ran `anvil`, did some transactions, got the `Computed Block Hash` and `rlp` using `eth-rlp-verify` (which is different from the block hash shown in anvil) and the `mmr proof` using `headers-accumulators`.

Reply:
What about the stamp rate proofs? Are those the one la from the ipynb or are also from Anvil?

Reply:
storage proof?